src/components/TextInput/TextInput.tsx

```diff
@@ -46,28 +46,48 @@ const TextInput = (props: Props) => {
     disabled,
     isInvalid,
     isValid,
+    validFeedback,
+    invalidFeedback,
     value,
     size,
     className,
     style,
   } = props
+  const inputDefaultStyles = {
+    borderColor: isValid ? '#59b571' : isInvalid ? 'red' : 'black',
+  }
+  const validFeedbackDefaultStyle = {
+    color: '#59b571',
+  }
```

Comment:
I see this logic duplicated in several different places. 

Can you explain why we need to have custom colors rather than using the default bootstrap colors?

Reply:
> I see this logic duplicated in several different places.
> 
> Can you explain why we need to have custom colors rather than using the default bootstrap colors?

 Hey @jackcmeyer thanks for the reply!
Sorry for all the duplicated logic considering the colors and styling, I try to improve and leave customized the styling whereever possible so I might've overdone it and strayed away from the bootstrap style and the projects guidelines. 
I will remove all of these custom styling and resolve the other requests asap!